blog.html

```diff
@@ -0,0 +1,350 @@
+          <p class="text-sm text-gray-600">21/09/2023<span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
+          </div> 
+        </div>
+      </div>
+      <hr style="height: 4px; background-color: orange; border: none;">
+    </div>
+
+    <div style="margin-bottom: 50px;"></div>
+    <!-- Popular Posts section -->
+    <aside class="popular-posts">
+      <h4 class="text-l font-semibold mb-4">Popular Posts</h4>
+      <ul>
+        <li class="flex items-start py-4">
+          <img src="assets\images\arthur-poulin-NhU0nUR7920-unsplash.jpg" alt="Popular Post 1" class="w-12 h-12 rounded mr-4">
+          <div>
+            <a href="/blog.html">
+            <span class="text-12px font-semibold">Traveling with Kids: Essential Accessories to Keep Your Little Ones Happy</span></a>
+            <br>
+            <br>
+          </div>  
+        </li>
+        <li class="flex items-start py-4">
+          <img src="assets\images\marissa-grootes-D4jRahaUaIc-unsplash 1.png" alt="Popular Post 2" class="w-12 rounded h-12 mr-4">
+          <div>
+            <a href="/blog.html">
+            <span class="text-12px font-semibold">Stay Organized on the Go: Best Travel Accessories</span></a>
+            <br>
+            <br>
+          </div>
+        </li>
+        <li class="flex items-start py-4">
+          <img src="assets\images\felix-rostig-UmV2wr-Vbq8-unsplash.jpg" alt="Popular Post 3" class="w-12 h-12 rounded mr-4">
+          <div>
+            <a href="/blog.html">
+            <span class="text-12px font-semibold">Top 10 Must-Have Travel Accessories for Your Next Adventure</span></a>
+            <br>
+            <br>
+          </div>
+        </li>
+        <li class="flex items-start py-4">
+          <img src="assets\images\tiffany-nutt-0ClfreiNppM-unsplash.jpg" alt="Popular Post 4" class="w-12 h-12 rounded mr-4">
+          <div>
+            <a href="/blog.html">
+            <span class="text-12px font-semibold">Eco-Friendly Travel: Sustainable Accessories for Responsible Explorers</span></a>
+            <br>
+            <br>
+          </div>
+        </li>
+      </ul>      
+    </aside>
+    </div>
+    <br>
+    <br>
+    <div class="button-container">
+      <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>Load More Posts »</b></button>
+    </div>
```

Comment:
<br> not best practice, better to use margins top/bottom to create space

<aside class="popular-posts"> would recommend not using this, check newsletter section on index.html or also how you have made the blog entries. Will remove having to make responsive you .popular.post styling

This section along with the blog posts should be in a 2 column grid that turns into a 1 column grid on small screens, check main branch code for tips on how to make responsive (All section are responsive on main now so can copy/paste what you need)

Centre images with text

Reply:
hasn't let me include the code in here, for the not best practice that is using `<br>`
wouldn't recomment using `<aside class="popular-posts">`